backend/homework/1/app/src/main/java/org/handson/Main.java

```diff
@@ -0,0 +1,90 @@
+package org.handson;
+
+
+
+public class Main {
+    Student getStudentDetail() {
+
+        Student student = new Student();
+
+
+        int id = 1;
+        student.setID(id);
+
+        String name = "mathangi";
+        student.setName(name);
+
+
+        int age = 21;
+        student.setAge(age);
+
+
+        char letter = 'A';
+        student.setGrade(letter);
+        return student;
+
+    }
+
+    void update(StudentRepo studentRepoobj) {
+
+
+        int id = 1;
+        Student student = studentRepoobj.getStudent(id);
+        if (student == null) {
+            System.out.println("User not found");
+            return;
+        }
+        int newage = 45;
+        studentRepoobj.updateStudentAge(newage, student);
+
+        char letter = 'B';
+        studentRepoobj.updateStudentGrade(letter, student);
+
+
+    }
+
+    void retrieve(StudentRepo studentRepoObj) {
+
+
+        int id = 1;
+        Student st1 = studentRepoObj.getStudent(id);
+        if (st1 != null)
+            CustomLogging.getLoggerCustom().debug("\nID: {} Name {} Grade {}  Age {}" ,st1.getID(), st1.getName() , st1.getGrade() , st1.getAge());
+
+        else
+            CustomLogging.getLoggerCustom().error("Unable to find user");
+
+
+        String name = "mathangi";
+        Student st = studentRepoObj.getStudent(name);
+        if (st != null)
+            CustomLogging.getLoggerCustom().info("\nID: {} Name {} Grade {} Age {}" ,st.getID(), st.getName() , st.getGrade(),st.getAge());
+        else
+            CustomLogging.getLoggerCustom().error("Unable to find user");
```

Comment:
Instead of `CustomLogging.getLoggerCustom()` just export this whole thing as `Logger`